gdal/doc/source/_extensions/driverproperties.py

```diff
@@ -270,6 +284,26 @@ def run(self):
         return finish_directive(self, 'supports_multidimensional', node)
 
 
+class DeprecatedDriverDirective(Directive):
+
+    # this enables content in the directive
+    has_content = True
+
+    def run(self):
+
+        env_variable = self.content[0]
+        self.content = docutils.statemachine.StringList(["This driver is considered for removal in GDAL 3.5. You are invited " \
```

Comment:
couldn't we make the version a parameter of the directive too ? I guess we'll also have to deprecate other drivers after 3.5. Otherwise looks good !

Reply:
Ok, see what you think of the update...

Reply:
excellent!